apps/api/src/deployment/services/top-up-managed-deployments/top-up-managed-deployments.service.ts

```
@@ -176,6 +191,7 @@ export class TopUpManagedDeploymentsService {
 
       this.summarizer.inc("deploymentTopUpErrorCount", ownerInputs.length);
       this.summarizer.trackFailedWallet(owner);
+      ownerInputs.forEach(() => this.instrumentation.recordDepositError("tx_error"));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)